### ISSUE TITLE
[Worklists/Twitter] Close shortcut not working on Twitter worklists

### ISSUE THREAD

**Issue by jstanden** [jstanden/cerb#1284]
This is likely from the security hardening.

Attribute [@ksuther](https://github.com/ksuther)

**Comment by jstanden** [comment]
~~~
Index: plugins/wgm.twitter/api/App.php
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- plugins/wgm.twitter/api/App.php	(revision 981d9f5327ea8fd9a67f5c61ff06b3ae37bb87e4)
+++ plugins/wgm.twitter/api/App.php	(date 1588382943622)
@@ -138,7 +138,7 @@
 		@$view_id = DevblocksPlatform::importGPC($_POST['view_id'],'string');
 		@$row_ids = DevblocksPlatform::importGPC($_POST['row_id'],'array', []);
 		
-		$models = DAO_TimeTrackingEntry::getIds($row_ids);
+		$models = DAO_TwitterMessage::getIds($row_ids);
 		
 		// Privs
 		$models = array_intersect_key(
~~~

**Comment by jstanden** [comment]
Fixed in 9.5.6
